src/fides/api/service/connectors/saas_connector.py

```diff
@@ -786,6 +797,41 @@ def _invoke_masking_request_override(
             )
             raise FidesopsException(str(exc))
 
+    @staticmethod
+    def _invoke_consent_request_override(
+        override_function_name: str,
+        client: AuthenticatedClient,
+        policy: Policy,
+        privacy_request: PrivacyRequest,
+        query_config: SaaSQueryConfig,
+        secrets: Any,
+    ) -> bool:
+        """
+        Invokes the appropriate user-defined SaaS request override for consent requests.
+
+        Includes the necessary data preparations for override input
```

Comment:
nit: this looks like a c+p error, i don't think there's any data prep done here!